### ISSUE TITLE
NZ COVIDpass should be called NZ COVID Pass

### ISSUE THREAD

**Issue by noway** [vaxxnz/vaxxed-as-web#15]
[The spec](https://nzcp.covid19.health.nz/) spells it as NZ COVID Pass

**Comment by rafcontreras** [comment]
Good catch, this changes are to be done at [Localazy](https://localazy.com/p/vaxxed-as), I'll probably get to it on Monday after work.
